iree/compiler/Codegen/Common/LinalgBufferizePass.cpp

```diff
@@ -380,6 +381,21 @@ static SmallVector<Value> getTiedOperandsForLinalgOps(
   return tiedOperands;
 }
 
+static LogicalResult analyseLinalgExtOps(linalg_ext::LinalgExtOp op,
+                                         BufferizationPlan &plan) {
+  if (!op.hasTensorSemantics()) return success();
+  for (auto input : op.getInputOperands()) {
+    plan.insert(input->get());
+  }
+  for (auto output : op.getOutputOperands()) {
```

Comment:
It might be OK to tie together the `op.getOutputOperands()` with the corresponding `op.getInputOperands()` if the `out` tensor has only one use. This is what should be done for `LinalgOp`s as well, except for a what is done for operand fusion today.

Reply:
I add a comment here. I think we don't have to handle it in this PR, since we have limit LinalgExt ops and it's not possible to add tests. linalg_ext.sort does not take inputs. We can revisit it after we have use cases.